packages/expo-camera2/ios/EXCamera2/EXCamera2ViewManager.m

```diff
@@ -0,0 +1,177 @@
+// Copyright 2016-present 650 Industries. All rights reserved.
+
+#import <EXCamera2/EXCamera2ViewManager.h>
+#import <EXCamera2/EXCamera2View.h>
+#import <UMCore/UMUIManager.h>
+
+#define EXECUTE_UI_BLOCK_WITH_SELF(block) \
+  UM_WEAKIFY(self); \
+  [_uiManager addUIBlock:^(id view){ \
```

Comment:
I think we should use `executeUIBlock` now

Reply:
Yeah, definitely!